setup.py

```diff
@@ -35,13 +35,19 @@ def load_requirements(use_case):
     },
     license="Apache2.0",
     classifiers=[
+        "Environment :: Console",
+        "Environment :: Web Environment",
         "Programming Language :: Python",
         "Programming Language :: Python :: 3",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
-        "Development Status :: 3 - Alpha",
+        "Programming Language :: Python :: 3.10",
```

Comment:
Should we add JS? `"Programming Language :: JavaScript",`

Reply:
I would opt no.

Reply:
The main logic is not in JavaScript. These should be about main items.